cmake/cmake-utilities.cmake

```diff
@@ -40,6 +40,8 @@ endfunction()
 function(make_utility name srcs)
     make_executable(${name} "${srcs}" "${CSOUNDLIB}")
     add_dependencies(${name} ${CSOUNDLIB})
```

Comment:
I don't think this is needed in modern cmake. It should be enough to target_link_libraries libcsound

Reply:
Ok, well, it's not needed either way so I deleted it. But we aren't allowed to `target_link_libraries` instead, I guess? https://github.com/csound/csound/pull/1748#issuecomment-1740207758